docs/reference/filter_field_definition.rst

```diff
@@ -192,6 +192,8 @@ Empty
         }
     }
 
+This filter has an additional option ``inverse`` in order to be used as a ``NotEmptyFilter``.
```

Comment:
```suggestion
The ``inverse`` option can be used to filter values that are not empty.
```
BTW, we should add an example using this option.

Reply:
I made the change.

But I don't think an example would be useful

This option can be used like any other filter option.
```
final class PostAdmin extends AbstractAdmin
    {
        protected function configureDatagridFilters(DatagridMapper $datagridMapper)
        {
            $datagridMapper
                ->add('deleted', EmptyFilter::class, ['field_name' => 'deletedAt', 'inverse' => true]);
        }
    }
```